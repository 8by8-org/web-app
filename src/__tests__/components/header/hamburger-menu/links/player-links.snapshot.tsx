import { render, cleanup } from '@testing-library/react'
import PlayerLinks from '@/components/header/hamburger-menu/links/player-links';

describe('PlayerLinks', () => {
  afterEach(cleanup);

  it('renders player links unchanged', () => {
    const { container } = render(<PlayerLinks />);
    expect(container).toMatchSnapshot();
  });
});
