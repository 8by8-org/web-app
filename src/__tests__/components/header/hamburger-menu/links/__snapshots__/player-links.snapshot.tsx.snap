// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerLinks renders player links unchanged 1`] = `
<div>
  <li>
    <a
      class="take_the_challenge_btn"
      href="/challengerwelcome"
    >
      Take The Challenge
    </a>
  </li>
  <li>
    <a
      class="link_lg_top"
      href="/actions"
    >
      Take Action
    </a>
  </li>
  <li>
    <a
      class="link_lg"
      href="/why8by8"
    >
      Why 8by8
    </a>
  </li>
  <li>
    <a
      class="link_lg"
      href="/rewards"
    >
      Rewards
    </a>
  </li>
  <li>
    <a
      class="link_lg"
      href="/faq"
    >
      FAQS
    </a>
  </li>
  <li>
    <a
      class="link_sm_top"
      href="/privacy-policy"
    >
      Privacy Policy
    </a>
  </li>
  <li>
    <a
      class="link_sm"
      href="/settings"
    >
      Settings
    </a>
  </li>
  <li>
    <button
      class="signout_btn"
    >
      Sign out
    </button>
  </li>
</div>
`;
